fabric-gamerule-api-v1/build.gradle

```diff
@@ -0,0 +1,16 @@
+archivesBaseName = "fabric-gamerule-api-v1"
```

Comment:
Should this be `fabric-game-rule-api-v1` or `fabric-gamerule-api-v1`?
Referring to #575

Reply:
Was brought up in previous pr, mojang refers to them as `Gamerules`
https://github.com/FabricMC/fabric/pull/283#issuecomment-519794948

Reply:
Alrighty

Reply:
I mean, that reasoning is not that good. Mojang calls them "game rules" (with two words) in more recent snapshot blog posts like [19w36a](https://www.minecraft.net/en-us/article/minecraft-snapshot-19w36a) and [20w17a](https://www.minecraft.net/en-us/article/minecraft-snapshot-20w17a).

Reply:
What does it say on the button for changing gamerules before world creation? because imo if it has both "game" and "rule" on the button then we should use however that's formatted.

Reply:
At least in the snapshot it was added in, it said `Game rules`.

Reply:
Spaced in the screen. So it's inconsistent of course.

Reply:
I wonder if we're going to need to use the proguard exports for internal reference purposes for this...

Reply:
I think `fabric-game-rule-api` would be fine since the spaced spelling is used by Yarn, in-game and in Mojang's most recent game rule-related articles.

Reply:
Yeah it seems they've moved to using separate words for Game Rules.